hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/balancer/BalancingPolicy.java

```diff
@@ -28,7 +28,7 @@
  * Balancing policy.
  * Since a datanode may contain multiple block pools,
  * {@link Pool} implies {@link Node}
- * but NOT the other way around
+ * but not the other way around
```

Comment:
I think this is just changing the case of Not, this much we can tolerate and feels like it was intentionally done to increase the weight of not or bring into notice. this isn't a typo nor a grammatical error by my books, just a way of writing which in general world is acceptable.

Reply:
OK
I think I learned something new.
Thanks @asymc for the guidance.
I will update the status of jira later.